Framework/Built_In_Automation/Web/Selenium/BuiltInFunctions.py

```diff
@@ -4570,7 +4570,18 @@ def upload_file_through_window(step_data):
         # elif platform.system() == "Linux":
         #     _gui_upload(path_name)
         else:
-            _gui_upload(path_name[0:-1])
+            #_gui_upload(path_name[0:-1])
```

Comment:
You commented out the `_gui_upload()` part but that's not that intended behavior.
@Arifshariar7 @Towshik I asked to implement this as an additional optional parameter:
```
use send keys | optional parameter | true
```